### ISSUE TITLE
Position Level Risk: add currency behind "Total portfolio VaR"

### ISSUE THREAD

**Issue by PavlaSip** [Deutsche-Boerse-Risk/DAVe-UI#40]
@markadir @hauehan @iyigmel: should we add currency behind the VaR number? To me it seems a bit unclear what the number should be: 
![positionlevelrisksnapshot](https://user-images.githubusercontent.com/30694014/28968802-140ce1c4-7922-11e7-9ff4-fc38696c1989.PNG)

**Comment by markadir** [comment]
Yes, let's add "EUR" after the number.

**Comment by Eddman** [comment]
@markadir @PavlaSip Probably related to #22.

I have the following data structure from API service:
```typescript
interface PositionReportServerData {
    clearingCurrency: string;
    productCurrency: string;
    compVar: number; // maps to Position VaR
    ... Other attributes ...
}
```
The bubble radius and `Total portfolio VaR` is calculated as a sum of `compVar` of all positions that are aggregated into single bubble and/or `Total portfolio VaR`.

Now the question is: what is the currency of `compVar`? Is it the same currency for all positions?

If the answer for the second question is "no" then we have a bug here. We can't create a plain sum of values that are in EUR, CHF and CZK at the same time... WDYT?

**Comment by PavlaSip** [comment]
@Eddman I would have assumed that the positions VaR would be in Clearing currency so same also for Total VaR, however it would be better if @markadir confirms. 
Also, I have a question @markadir : can NCMs belonging to one CM have a different clearing currency than the CM? If so, then if the user is a CM and sees the simple sum of his NCMs (each with a different Clearing currency) this would also be a bug. 
Also, this principle of simply summing up  then also applies to the Margin Requirement Overview for the totals in the green boxes?

**Comment by Eddman** [comment]
@PavlaSip For all aggregations we use a simple sum. In #23 I've already referenced the aggregation table on Dashboard where we just do a plain sub, but the currencies are obviously different.

**Comment by Eddman** [comment]
Based on what @orsamir told me the `compVar` is always in clearing currency and there is a single clearing currency for a member. So all rows have the same currency and this sum should be correct.

However we have problems on other views. I've created a new issue (#42) to discuss the aggregation problems there.

Here I would say we just add the clearing currency to the values and close the issue.

**Comment by markadir** [comment]
@PavlaSip : It is planned that different pools can have different currencies but NCMs separately can not have a different Clearing currency then pool. I think we should always differenciate at least on pool Level. Hence the latest Aggregation node where we are allowed to sum something should be the pool.

**Comment by PavlaSip** [comment]
@markadir Yes, agree. I have also already discussed with some colleagues and noticed that we never aggregate higher than pool level - even in the member report CI060 it ends at total per pool. Would that mean we need to change the UI - remove the totals from Position and Margin Requirement Overview and perhaps display sum of Margin Requirement and e.g. Collateral value in the "Pool Margin" view?

**Comment by markadir** [comment]
The dashboard change should be covered in [Issue 33](https://github.com/Deutsche-Boerse-Risk/DAVe-UI/issues/33). We don't sum margin requirements over pools. Position View is anyways split down to positions and ComVar views I think are also on account Levels. However the Topic should not be discussed here since we polute issues with unrelated Topics.

**Comment by orsamir** [comment]
In the current Prisma, compVar (as well as the Initial margin) is always in the clearer's clearing currency, even if pool reporting currency is different.

**Comment by markadir** [comment]
So as we now understood it's in Clearing Currency hence FX has been applied to comVar numbers already by Prisma. We don't need to take care of the aggregation of different currencies. Also wenn pools will have different currencies its not a problem since this graphic is aggregated already way below on account Level. We should just display it, so it is clear. You should at least in the summary sentence say: ` "Total Portfolio Var is " +string(compVar) + clearingCurrency`

**Comment by Eddman** [comment]
@PavlaSip, @markadir Please verify and close the issue.

**Comment by PavlaSip** [comment]
Looks good, thanks.
